Picker/Picker.js

```diff
@@ -116,108 +72,71 @@ const PickerBase = kind({
 		value: 0
 	},
 
-	styles: {
-		css,
-		className: 'picker',
-		publicClassNames: true
-	},
-
-	handlers: {
-		handleDecrement: decrement,
-		handleFlick: handle(
-			forEventProp('direction', 'vertical'),
-			// ignore "slow" flicks by filtering out velocity below a threshold
-			oneOf(
-				[({velocityY}) => velocityY < 0, increment],
-				[({velocityY}) => velocityY > 0, decrement]
-			)
-		),
-		handleIncrement: increment
-	},
-
 	computed: {
-		activeClassName: ({styler}) => styler.join('active', 'item'),
-		decrementAriaLabel: ({'aria-valuetext': valueText, children: values, decrementAriaLabel = $L('previous item'), value}) => {
-			return `${valueText != null ? valueText : values[value]} ${decrementAriaLabel}`;
-		},
-		valueId: ({id}) => `${id}_value`,
-		incrementAriaLabel: ({'aria-valuetext': valueText, children: values, incrementAriaLabel = $L('next item'), value}) => {
-			return `${valueText != null ? valueText : values[value]} ${incrementAriaLabel}`;
+		disabled: ({children, disabled}) => React.Children.count(children) > 1 ? disabled : true,
+		max: ({children}) => children && children.length ? children.length - 1 : 0,
+		value: ({value, children}) => {
+			const max = children && children.length ? children.length - 1 : 0;
+			return clamp(0, max, value);
 		}
 	},
 
 	render: (props) => {
-		const {
-			'aria-valuetext': valueText,
-			activeClassName,
-			children: values,
-			decrementAriaLabel,
-			handleDecrement,
-			handleFlick,
-			handleIncrement,
-			incrementAriaLabel,
-			value,
-			valueId,
-			...rest
-		} = props;
-		const isFirst = value <= 0;
-		const isLast = value >= React.Children.count(values) - 1;
+		const {children, value, max, ...rest} = props;
```

Comment:
We usually define `prop`s alphabetically like `{children, max, value, ...rest}`.